README.md

```diff
@@ -22,9 +39,275 @@ Or install it yourself as:
 
     $ gem install accessly
 
+Add the ActiveRecord Migrations:
+
+    $ rails g accessly:install
+
 ## Usage
 
-TODO: Write usage instructions here
+You can use the Accessly gem directly to grant | revoke | check permissions.  We recommend the use of 'Policies' covered in this README.
+Checkout our [API docs](http://www.rubydoc.info/gems/accessly) for more info on using the API directly
+
+We use Accessly with policies in mind to capture everything we want to know about a specific permission set. Let's take a look at some examples:
+
+### Basic Action Policy
+
+```ruby
+class ApplicationFeaturePolicy < Accessly::Policy::Base
+
+  actions(
+    view_dashboard: 1,
+    view_super_secret_page: 2,
+    view_double_secret_probation_page: 3
+  )
+
+end
+```
+
+With this policy we can `grant` permissions to a user
+
+```ruby
+ApplicationFeaturePolicy.new(user).grant!(:view_super_secret_page)
```

Comment:
Through some oversight by some developer who didn't know what he was doing, we defined `grant` and `revoke` without the `!`.

Either we should add the `!` to those methods, or remove it here.

Reply:
:( - let's add them - I'll open a bug